tasks/configure.yml

```diff
@@ -0,0 +1,47 @@
+# tasks file for syncthing
+---
+- name: configure | ensure directory exists
+  file:
+    path: "{{ syncthing_system_user_home }}/.config/syncthing"
+    state: directory
+    owner: "{{ syncthing_system_user }}"
+    group: "{{ syncthing_system_group }}"
+    mode: 0755
+    recurse: yes
+  tags:
+    - syncthing-configure-directories
+
+# - debug: var=hostvars[inventory_hostname]
+# - debug: var=groups
```

Comment:
I prefer removing comment; since if you know ansible you already know how to debug it